src/modules/governance/ProposalsList.tsx

```diff
@@ -88,7 +88,7 @@ export function ProposalsList({ proposals: initialProposals }: GovernancePagePro
   }
 
   usePolling(fetchNewProposals, 60000, false, [proposals.length]);
-  usePolling(updatePendingProposals, 15000, false, [proposals.length]);
+  usePolling(updatePendingProposals, 30000, false, [proposals.length]);
```

Comment:
there's around 100 votes in 5 days... seems a bit unreasonable to refetch every 15s, even 30 seems unreasonable, but i guess small steps are better suited to test if someone complains.